cypress/integration/Toggle.spec.js

```diff
@@ -0,0 +1,91 @@
+/// <reference types="Cypress" />
+
+/* eslint-disable no-undef */
```

Comment:
You can add an `.estlinrc` in the `/cypress` folder that disables that rule, that way it doesn't need to be added in each file. I only learned this recently - probably would have included it in the initial Cypress setup otherwise.